cabra/src/pages/components/SelectBox.tsx

```diff
@@ -0,0 +1,39 @@
+import Select, { SingleValue } from "react-select";
+
+import { useState } from "react";
+
+export interface PaginationOption {
+  readonly value: number;
+  readonly label: string;
+}
+
+const paginationOptions: PaginationOption[] = [
+  { value: 1, label: "1" },
+  { value: 2, label: "2" },
+  { value: 5, label: "5" },
+  { value: 10, label: "10" },
+  { value: 20, label: "20" },
+  { value: 100, label: "100" },
+];
+
+export default function SelectBox(props: {
+  parentCallback: (arg0: number) => void;
+}) {
+  const [selectedOption, setSelectedOption] =
+    useState<PaginationOption | null>();
```

Comment:
If you don't pass the argument, the value will be undefined and the React may spam with warnings in the console
```suggestion
  const [selectedOption, setSelectedOption] =
    useState<PaginationOption | null>(null);
```